packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -67,13 +67,10 @@ export class AnchorMenuItem extends AnchorBase {
         return true;
     }
 
-    // The FAST Menu manages the `tabindex` of its menu items. Because of a bug in Chromium
-    // (https://bugs.chromium.org/p/chromium/issues/detail?id=1346606), setting the tabindex on an element
-    // with `delegatesFocus=true` causes the element to lose focus. This causes the menu to close, preventing
-    // arrowing through the items or navigating to the url. As a workaround, we intercept attempts to
-    // set the tabindex on the host and instead set it on the inner anchor.
-    // The referenced Chromium bug is actually fixed, but it hasn't been pulled into Edge yet (it is in
-    // Chrome). Issue https://github.com/ni/nimble/issues/1118 tracks removal of this workaround.
+    // The FAST Menu manages the `tabindex` of its menu items.
```

Comment:
Actually wait, just to be clear, does this manifest in an issue across all browsers or is it browser specific? I thought the arrow traversal issue was only in Edge?

Reply:
Oh sorry the other thread discussed that it is across browser.